binaries/dump_operator_names.cc

```diff
@@ -29,7 +29,8 @@ void dump_opnames(const Module& m, std::unordered_set<std::string>& opnames) {
   for (const auto& method : methods) {
     const auto& func = method.function();
     std::cout << "function name: " << func.name() << std::endl;
-    torch::jit::Code code(func.graph(), "");
+    auto graph = toGraphFunction(func).graph()->copy();
+    torch::jit::Code code(graph, "");
```

Comment:
Since you are working with stacks anyway, can you perhaps move this change to a separate PR?

Reply:
@malfet  Moved this change to https://github.com/pytorch/pytorch/pull/71246